pkg/scheduler/preemption/preemption.go

```diff
@@ -63,6 +63,14 @@ type Preemptor struct {
 	applyPreemption func(ctx context.Context, w *kueue.Workload, reason, message string) error
 }
 
+type preemptionCtx struct {
+	log               logr.Logger
+	wl                workload.Info
+	snapshot          *cache.Snapshot
+	frsNeedPreemption sets.Set[resources.FlavorResource]
+	quantities        resources.FlavorResourceQuantities
```

Comment:
a less ambiguous name. perhaps workload requests?

Reply:
Right, the name is ambiguous, but workload requests might be misleading, because in some contexts (like the oracle) the requests are scoped to only one flavor, resource pair. 

Maybe `requestsNeedPreemption` by analogy to `frsNeedPreemption`?

Reply:
Is ```requestsNeedPreemption``` accurate? Doesn't this include non-preempt FRs?

Reply:
You are right actually, it probably does not need to include the resources non-requiring preemption.
So maybe `requests` isn't bad?